examples/minimal-appdir/src/server/uploadthing.ts

```diff
@@ -26,7 +26,7 @@ export const uploadRouter = {
       acl: "public-read",
     },
     video: {
-      maxFileSize: "16MB",
```

Comment:
revert?

Reply:
yea i just didn't have any file between 5 and 16 megs to test with haha